chain/telemetry/src/lib.rs

```diff
@@ -1,8 +1,9 @@
 use std::time::Duration;
+use std::u64;
```

Comment:
There is no need to do this. `u64::max_value()` is available just fine without this `use`.

Reply:
Good point, fixed now. I'm confusing that why `std::u64::max_value()` => compile error. why it's not equivalent to `use std::u64; u64::max_value()`? 
(u64::max_value() is just fine without this use too, as you said)